app/models/ncr/work_order.rb

```diff
@@ -193,19 +193,55 @@ def system_approver_emails
     end
 
     def self.ba61_tier1_budget_mailbox
-      ENV['NCR_BA61_TIER1_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com'
+      users = User.with_role('BA61_tier1_budget_approver').where(client_slug: 'ncr')
+      if !users or users.length == 0
+        user = User.with_email_role_slug!(
+          ENV['NCR_BA61_TIER1_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
```

Comment:
In other words, creating the record in this method doesn't *quite* feel right. Having required records in databases is strange to me...hence my previous leanings to environment variables :wink: